src/game/logic/object/update/aiupdate.h

```diff
@@ -176,6 +193,27 @@ class AICommandInterface
         AICommandParms params(AICMD_IDLE, cmd_source);
         AI_Do_Command(&params);
     }
+
+    void AI_Enter(Object *obj, CommandSourceType cmd_source)
+    {
+        AICommandParms params(AICMD_ENTER, cmd_source);
+        params.m_obj = obj;
+        AI_Do_Command(&params);
+    }
+
+    void AI_Evacuate(bool b, CommandSourceType cmd_source)

```

Comment:
bool for_attack
or
bool reenter

The AssaultTransport has this feature where transported units will temporarily evacuate to attack a target, and once the attack is completed, they will automatically reenter the vehicle.

Reply:
fixed.